retriever/lib/scripts.py

```diff
@@ -18,6 +18,7 @@
 from retriever.lib.defaults import SCRIPT_SEARCH_PATHS, VERSION, ENCODING, SCRIPT_WRITE_PATH
 from retriever.lib.load_json import read_json
 
+global_temp_scripts = {}
```

Comment:
So I had the temp in there to try and convey the fact that the scripts compiled are temporary to the instance of the program run and not cached elsewhere in a file or something like that. I can see how it can be a bit confounding though, I can change it to global_script_list, I think that'd fit better.